src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -270,10 +273,15 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('command.xml');
 
         if ('auto' === $config['adapter']) {
-            $container->setAlias(
-                'sulu_media.adapter',
-                'sulu_media.adapter.' . (class_exists('Imagick') ? 'imagick' : 'gd')
-            );
+            $adapter = 'gd';
+            if (class_exists('Imagine\Vips\Imagine')) {
```

Comment:
Any reason not to use the `::class` constant`

Reply:
yes - phpstan will throw an error  in the CI/CD (https://travis-ci.org/sulu/sulu/jobs/634123652#L1272) and we cannot install the lib - because then we would have to install the extension `vips`

Reply:
If its just phpstan I would add it to ignore errors.